app.js

```diff
@@ -133,50 +133,55 @@ const tslint = {
       default: 'latest',
       filter: t => {
         return t === '5' ? '1' : t;
-      },
+      }
     },
     {
       type: 'text',
       name: 'name',
       message: 'Project Name? 🤔 (non alpha characters will be stripped out)\n',
       filter: t => {
-        return t.replace(/[^a-zA-Z]/g, '');
-      },
+        return t.replace(/[^a-zA-Z-]/g, '');
+      }
+    },
+    {
+      type: 'text',
+      name: 'path',
+      message: 'Do you want a specific path? if so please specify it: \n'
```

Comment:
Typo and missing an emoji to fit with the other questions.

Please use: https://www.npmjs.com/package/inquirer-select-directory
to select a path.  That way, no bug from the end-user can happen in the path selection.